CONTRIBUTING.md

```diff
@@ -57,9 +63,9 @@ Prior to opening a pull request, ensure that your code has been auto-formatted,
 and that it passes both the lint and test validation checks:
 
 ```shell
-cargo fmt  # Auto-formatting...
-cargo test  # Testing...
```

Comment:
Why remove formatting and testing here?

Reply:
formatting is done by pre-commit, testing just moved below clippy